pkg/apis/postgres-operator.crunchydata.com/v1beta1/shared_types.go

```diff
@@ -366,6 +377,8 @@ func (in *AdditionalVolume) AsVolume(name string) corev1.Volume {
 			ClaimName: in.ClaimName,
 			ReadOnly:  in.ReadOnly,
 		}
+	case in.Image != nil:
+		out.Image = in.Image
```

Comment:
❓ The description on this method says it returns a copy. Does `corev1.ImageVolumeSource` have a `DeepCopy` method?

```suggestion
		out.Image = in.Image.DeepCopy()
```